components/dashboard/src/menu/Menu.tsx

```diff
@@ -0,0 +1,221 @@
+                (showUsageView || (teamBillingMode && teamBillingMode.mode === "usage-based"))
+            ) {
+                leftMenu.push({
+                    title: "Usage",
+                    link: `/usage`,
+                });
+            }
+            if (currentUserInTeam?.role === "owner") {
+                leftMenu.push({
+                    title: "Settings",
+                    link: `/settings`,
+                    alternatives: getTeamSettingsMenu({
+                        team,
+                        billingMode: teamBillingMode,
+                        ssoEnabled: oidcServiceEnabled,
+                    }).flatMap((e) => e.link),
+                });
+            }
+        }
+        return leftMenu;
+    }, [oidcServiceEnabled, showUsageView, team, teamBillingMode, teamMembers, user, userBillingMode]);
+
+    const handleFeedbackFormClick = () => {
+        setFeedbackFormVisible(true);
+    };
+
+    const onFeedbackFormClose = () => {
+        setFeedbackFormVisible(false);
+    };
+
+    return (
+        <>
+            <header className="app-container flex flex-col pt-4 space-y-4" data-analytics='{"button_type":"menu"}'>
+                <div className="flex h-10 mb-3">
+                    <div className="flex justify-between items-center pr-3">
+                        <Link to="/" className="pr-3">
+                            <img src={gitpodIcon} className="h-6" alt="Gitpod's logo" />
+                        </Link>
+                        <OrganizationSelector />
+                        {!isMinimalUI && (
+                            <>
+                                <div className="pl-2 text-base text-gray-500 dark:text-gray-400 flex">
+                                    {leftMenu.map((entry) => (
+                                        <div className="p-1" key={entry.title}>
+                                            <PillMenuItem
+                                                name={entry.title}
+                                                selected={isSelected(entry, location)}
+                                                link={entry.link}
+                                            />
+                                        </div>
+                                    ))}
+                                </div>
+                            </>
+                        )}
+                    </div>
+                    <div className="flex-1 flex items-center w-auto" id="menu">
+                        <nav className="flex-1">
+                            <ul className="flex flex-1 items-center justify-between text-base text-gray-500 dark:text-gray-400 space-x-2">
+                                <li className="flex-1"></li>
+                                {user?.rolesOrPermissions?.includes("admin") && (
+                                    <li className="cursor-pointer">
+                                        <PillMenuItem name="Admin" link="/admin" />
```

Comment:
**thought**: Do know if this was by accident but now that the admin menu is not highlighted when active we could also resolve https://github.com/gitpod-io/gitpod/issues/15012, as long as we update the header to have a fixed title for the admin. Later on, moving this to the user menu could be better.

Reply:
I'll change this in this PR if there are other things that need to be addressed. If not, let's fix it afterward.

Reply:
Opened follow-up https://github.com/gitpod-io/gitpod/issues/16238 to keep track of this. Cc @svenefftinge